src/chain/index.ts

```diff
@@ -60,7 +62,8 @@ export class BeaconChain extends EventEmitter {
   /**
    * Initialize the beacon chain with a genesis beacon state / block
    */
-  public async initializeChain(genesisTime: number64, genesisDeposits: Deposit[], genesisEth1Data: Eth1Data): Promise<void> {
+  public async initializeChain(genesisTime: number64, genesisDeposits: Deposit[]
```

Comment:
I'd prefer if everything was on its own line for constructors
```suggestion
  public async initializeChain(
    genesisTime: number64, 
    genesisDeposits: Deposit[],
```